### DISCUSSION TITLE
delegate나 event 키워드는 ZepetoScript에서 어떻게 사용하나요?

### DISCUSSION THREAD

**Discussion by mayko1234** [naverz/zepeto-studio-kor#343] [5. Q&A]
안녕하세요, 
유니티 C#에서 사용하는 delegate나 event 키워드는 ZepetoScript에서 어떻게 사용하는 지 알고싶습니다. 
아시는 분 계시면 답변 부탁드려요~!

**Comment by pumfit** [comment]
ZepetoScript는 typescript를 사용하는데 typescript에는 C#처럼 delegate,event 키워드가 존재하지 않는 걸로 알고 있습니다.

대신 제페토스크립트에서는 Unity에서 제공하는 UnityEvent( C#의 이벤트와 델리게이트(Events and Delegates)를 사용하기 쉽게 유니티에서 제공하는 것 )를 제공하고 있습니다.

아래와 같이 공식 가이드에 관련 내용이 있으니 예제를 보시는걸 추천 드립니다! 
https://studio.zepeto.me/kr/guides/events
